cmd/osm-bootstrap/osm-bootstrap.go

```diff
@@ -429,7 +429,40 @@ func (b *bootstrap) createMeshRootCertificate() error {
 	}
 
 	createdMRC.Status = configv1alpha2.MeshRootCertificateStatus{
-		State: constants.MRCStateActive,
+		State: constants.MRCStatePending,
```

Comment:
For the default MRC created on start-up should the state be active?

Reply:
I'm not 100% sure on that; wouldn't we want the controller to verify that the MRC is active first? What if the issuer can't be contacted?

Reply:
would the conditions also be known in this case?

Reply:
For the preset MRC, we may still have to connect to the issuer (a k8s secret, cert-manager, vault, etc.) and that connection may fail; wouldn't the MRC be waiting for a controller to confirm things are working as expected (i.e. pending) rather than assuming everything's working during creation?

Reply:
That makes sense to me! I think I had this field confused with `intent` in the MRC spec.